native/SteamWrap.cpp

```diff
@@ -399,18 +399,36 @@ static bool CheckInit()
 }
 
 //-----------------------------------------------------------------------------------------------------------
-value SteamWrap_Init(value onEvent)
+value SteamWrap_Init(value onEvent, value notificationPosition)
 {
 	bool result = SteamAPI_Init();
 	if (result)
 	{
 		g_eventHandler = new AutoGCRoot(onEvent);
 		s_callbackHandler = new CallbackHandler();
-		SteamUtils()->SetOverlayNotificationPosition( k_EPositionTopLeft );
+
+		switch (val_int(notificationPosition))
+		{
+		case 0:
+			SteamUtils()->SetOverlayNotificationPosition(k_EPositionTopLeft);
+			break;
+		case 1:
+			SteamUtils()->SetOverlayNotificationPosition(k_EPositionTopRight);
+			break;
+		case 2:
+			SteamUtils()->SetOverlayNotificationPosition(k_EPositionBottomRight);
+			break;
+		case 3:
+			SteamUtils()->SetOverlayNotificationPosition(k_EPositionBottomLeft);
+			break;
+		default:
+			SteamUtils()->SetOverlayNotificationPosition(k_EPositionBottomRight);
+			break;
+		}
```

Comment:
style note: if you could please indent the case statements and their bodies by one tab each so they start one tab in from the open & close brackets